docs/api/index.rst

```diff
@@ -12,6 +12,7 @@ QCoDeS API
    config
    dataset
    instrument
+   parameters
    logger
    math
    monitor
```

Comment:
shall the order here make sense? i see it's not alphabetical, but i can't see what the current logic is. i'd suggest to go from config through parameters and instruments to measurement and dataset, with monitor, logger and other stuff inbetween where relevant.

Reply:
That is why this is WIP